README.md

```diff
@@ -294,14 +242,25 @@ Get-AzADServicePrincipal -ServicePrincipalName $($app.ApplicationId.Guid) -OutVa
 } | Convertto-json
 #endregion
 
-#region create local environment variables
-[Environment]::SetEnvironmentVariable("AIPASClientId", "$($app.ApplicationId)", "User")
-[Environment]::SetEnvironmentVariable("AIPASClientSecret", "$PlainPassword", "User")
-[Environment]::SetEnvironmentVariable("AIPASSubscriptionId", "$($subscription.subscriptionId)", "User")
-[Environment]::SetEnvironmentVariable("AIPAStenantId", "$($subscription.TenantID)", "User")
-[Environment]::SetEnvironmentVariable("AIPASResourceGroupName", $ResourceGroupName, "User")
-[Environment]::SetEnvironmentVariable("AIPASStorageAccountName", $StorageAccountName, "User")
-# Restart VSCode to have access to the environment variables
+#region create local settings file
```

Comment:
Hi @rwaal

For running the Pester tests located in the IPAM PowerShell Module environment variables are used. So these still need to be created on your local development machine